client/src/test/updateTools.test.ts

```diff
@@ -0,0 +1,55 @@
+import * as assert from 'assert';
+import TestHelper, { LONG, SETUP_TIMEOUT, SIMPLE } from './TestHelper';
+
+suite('Viper Tools Update Test', () => {
+
+    suiteSetup(async function() {
+        this.timeout(SETUP_TIMEOUT);
+        await TestHelper.setup();
+        // these tests require a running backend:
+        await TestHelper.startExtension();
+    });
+
+    suiteTeardown(async function() {
+        await TestHelper.teardown();
+    });
+
+    test("Viper Tools Update Test & test abort of first verification", async function() {
+        this.timeout(60000);
```

Comment:
This seems quite optimistic. Just keep in mind that it can take 2-3 min on a mobile connection.